forte/mrdsrg-spin-adapted/sadsrg.cc

```diff
@@ -331,8 +333,14 @@ void SADSRG::init_fock() {
     local_timer lt;
     print_contents("Filling Fock matrix from ForteIntegrals");
     Fock_ = BTF_->build(tensor_type_, "Fock", {"gg"});
+
+    bool JKSafe = foptions_->get_bool("EMBEDDING_JKFOCK");
+
+    if (!JKSafe) {
```

Comment:
Why do we have code to _initialize_ Fock, and also code to _build_ Fock? What are the responsibilities of these two codes? The fact that both codes care about `EMBEDDING_JKFOCK` is a warning sign that they are tightly coupled.